README.md

```diff
@@ -1,2 +1,115 @@
+[![Build Status](https://travis-ci.com/siefkenj/tafs.svg?branch=master)](https://travis-ci.com/siefkenj/tafs)
+
+
 # tafs
 TA Feedback System
+
+## Initial Set Up
+
+```
+npm install
+```
+
+## Building
+
+```
+npm start
+```
+
+Note: This starts a PHP server on port 3000.
+
+## Database Setup
+
+Login to database via Mariadb (I used MYSQL client via windows)
+
+Once logged, do the following to create your local database and user to allow access via php:
+
+### 1. Database creation
+
+```
+MariaDB[(none)]> CREATE DATABASE `mydb`;
+```
+
+### 2. User creation
+
+```
+MariaDB[(none)]> CREATE USER 'myuser' IDENTIFIED BY 'mypassword';
+```
+
+### 3. Grant permissions to access and use the MySQL server
+
+Only allow access from localhost (this is the most secure and common configuration you will use for a web application):
+
+```
+MariaDB[(none)]> GRANT USAGE ON *.* TO 'myuser'@localhost IDENTIFIED BY 'mypassword';
+```
+
+To apply the changes of the new assigned permissions, you must finish with the following command:
+
+```
+MariaDB[(none)]> FLUSH PRIVILEGES;
+```
+
+### 5. Verify your new user has the right permissions
+
+```
+MariaDB[(none)]> SHOW GRANTS FOR 'myuser'@localhost;
++--------------------------------------------------------------+
+| Grants for myuser@localhost                                  |
++--------------------------------------------------------------+
+| GRANT USAGE ON *.* TO 'myuser'@'localhost'                   |
+| GRANT ALL PRIVILEGES ON `mydb`.* TO 'myuser'@'localhost'     |
++--------------------------------------------------------------+
+2 rows in set (0,00 sec)
+```
+
+If you made a mistake at some point you can undo all the steps above by executing the following commands, taking the precaution of replacing localhost with ‘%’ if you also changed it in the previous commands:
+
+```
+DROP USER myuser@localhost;
+DROP DATABASE mydb;
+```
+
+#### Now that the Database is created, you could use the following command to see the list of databases that are available
+
+```
+MariaDB[(none)]> SHOW DATABASES;
+```
+
+Select the database that's being used
+
+```
+MariaDB[(none)]> USE database_name;
+```
+
+Once the database is selected, import the db/schema.sql file
```

Comment:
The path `db/schema.sql` is inconsistent with #9. Probably that file should be moved (I like `db/schema.sql` better).

Reply:
That is changed in the updated pull request Darren submitted.